src/components/interface/SwapForm/components/Leverage/Leverage.tsx

```diff
@@ -124,56 +90,29 @@ const Leverage = ({
         sx={{
           flexGrow: '1',
           marginLeft: (theme) => theme.spacing(4),
+          marginTop: (theme) => theme.spacing(5.5),
           display: 'flex',
           alignItems: 'center',
+          columnGap: (theme) => theme.spacing(2),
+          width: "232px"
         }}
       >
-        <Slider
-          disabled={isDisabled}
-          min={low}
-          max={high}
-          step={0.01}
-          value={internalValue}
-          onChange={handleChangeSlider}
-          id={"Slider"}
-          onChangeCommitted={handleChangeCommittedSlider}
-          marks={[
-            {
-              value: low,
-              label: `${formatNumber(low)}x`,
-            },
-            {
-              value: rainbow1,
-              label: `${formatNumber(rainbow1)}x`,
-            },
-            {
-              value: high,
-              label: `${formatNumber(high)}x`,
-            },
-          ]}
-          sx={{
-            marginTop: '16px',
-            '& .MuiSlider-track': {
-              background: `linear-gradient(90deg, #00D395 ${rainbow2Percent}%, #F1D302 ${rainbow3Percent}%, #F61067 ${rainbow4Percent}%)`,
-              width: `${rainbowWidth}% !important`,
-              left: `${rainbowStart}% !important`,
-              display: isDisabled ? 'none' : undefined,
-            },
-            '& .MuiSlider-thumb': {
-              display: isDisabled ? 'none' : undefined,
-            },
-            '& .MuiSlider-mark[data-index="1"]': {
-              height: '6px',
-              width: '3px',
-              background: isDisabled ? undefined : colors.vzCustomGreen2.base,
-            },
-            '& .MuiSlider-mark[data-index="2"]': {
-              height: '6px',
-              width: '3px',
-              background: isDisabled ? undefined : colors.vzCustomRed2.base,
-            },
-          }}
-        />
+        {options.map(opt => {
+          return (
+            <Button
+              variant={"contained"}
```

Comment:
```suggestion
              key={opt.id}
              variant={"contained"}
```

add the key prop, required by React when enumerating a list

Reply:
Included the key prop as key={opt} since I removed the id variable and left it as an array of numbers